dev-libs/soci/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>WolfgangESanyer@Gmail.com</email>
+		<name>Wolfgang E. Sanyer</name>
+	</maintainer>
```

Comment:
You still need proxy-maint here or a gentoo dev

Reply:
See [here][1], I  think there may be some github shenanigans going on, or maybe I just hadn't pushed the update when you did this review.

[1]: https://github.com/gentoo/gentoo/pull/19865/files?file-filters%5B%5D=.xml#diff-f27b045bf2a2451cb68ad2bce7d4fdfda159683536d887ee4a2c7007ec509525R8